### DISCUSSION TITLE
METreadnc

### DISCUSSION THREAD

**Discussion by cristianastan2** [dtcenter/METplus#1633] [Existing Builds] [answered]
Hi,

I'm trying to run the RMM use_case on NOAA machine Hera and METreadnc is missing. I'm using the configuration 
listed for Hera on this website: https://dtcenter.org/community-code/metplus/metplus-4-1-existing-builds

Any help on what module needs to be added will be appreciated.

Thank you,
Cristiana

**Comment by hankenstein2** [comment]
Unfortunately the person we need, Julie Prestopnik, to answer this question
is out until Jun 6. I am not sure how Hera is setup.
METreadnc is found in METdatadb, as a guess could you try


   - module use /contrib/METdatadb/modulefiles
   - module load metdatadb/1.1.0


And see if that works?

Hank

On Wed, May 25, 2022 at 8:41 PM cristianastan2 ***@***.***>
wrote:

> Hi,
>
> I'm trying to run the RMM use_case on NOAA machine Hera and METreadnc is
> missing. I'm using the configuration
> listed for Hera on this website:
> https://dtcenter.org/community-code/metplus/metplus-4-1-existing-builds
>
> Any help on what module needs to be added will be appreciated.
>
> Thank you,
> Cristiana
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1633>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHAUIKH2YTXB6MDVVYJ2KGLVL3QHPANCNFSM5W7L5FPA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--------------------------------------------------------------
Hank Fisher       Research Applications Laboratory
303-497-2817      National Center for Atmospheric Research
***@***.***  P.O. Box 3000, Boulder CO 80307-3000
--------------------------------------------------------------

**Reply by CPKalb** [reply]
Hi Cristiana and Hank:

After hearing back from Julie, it sounds like METdatadb is not installed on Hera.  Hank, do you know if there is anyone else who could add this to Hera before Julie returns?

Christina

**Comment by CPKalb** [comment]
Hi Cristiana:

The METreadnc scrips are located in the the [METdatadb ](https://github.com/dtcenter/METdatadb) repository.  I'll need to double check with @jprestop to see if this is installed on Hera.  If it is installed, then it's probably not in your python path.

Thanks,
Christina

**Comment by CPKalb** [comment]
Hi Cristiana:

As an update, Julie does not have permission to create the METdatadb directory.  She has submitted a ticket to their help desk to try and get it installed but it will likely take some time.

Christina

**Reply by cristianastan2** [reply]
Thank you all. I appreciate the effort in helping me with this issue. 

Cristiana

**Comment by jprestop** [comment, marked as answer]
@cristianastan2 Thank you for your patience while I was out and also waiting on the directory creation.  I have installed METdatadb on Hera and have updated the Hera section of our [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-4-1-existing-builds) page with instructions on how to load the module to set up the environment.

I was able to successfully run the RMM use case with the following command:
```
run_metplus.py -c /contrib/METplus/METplus-4.1.1/parm/use_cases/model_applications/s2s/UserScript_obsERA_obsOnly_RMM.conf -c /scratch1/BMC/dtc/Julie.Prestopnik/METplus/hera.
jpresto.conf
```

Please give it a try and let us know if you have any problems.

**Reply by cristianastan2** [reply]
Thank you, @jprestop. After loading the METdatadb I'm also able to run the /parm/use_cases/model_applications/s2s/UserScript_obsERA_obsOnly_RMM.conf use case. 

I noticed that on the Existing Builds page for Hera there is a typo:

module use /contrib/MET**datab**/modulefiles -> datadb

Cristiana

**Reply by jprestop** [reply]
@cristianastan2   Thank you for confirming that after loading METdatadb you are also able to run the use case, for selecting an answer, and for letting me know that there was a typo.  I have corrected the typo on the Existing Builds page.

At this point I'm going to LOCK this conversation to prevent future posts. Our team has decided that once discussions have been answered, and the answer has been confirmed, we'll lock them. We want to encourage users to ask new questions in new discussions rather than posting to old ones. Hopefully that'll make the questions/answers easier for other users to follow. So if/when more issues/questions arise, please feel free to start a new discussion.
